content/05.main-text.md

```diff
@@ -412,7 +412,34 @@ we apply the following criteria to calculate _p~empiric~_ from summary statistic
 
 ### Rest API & backend
 
-<https://search-api.het.io/v1/>
+We created a backend application using Python's Django web framework.
+The source code is available in the [`connectivity-search-backend`](https://github.com/greenelab/connectivity-search-backend) repository.
+The primary role of the backend is manage a relational database and provide an API for requesting data.
+
+We define the database schema [using](https://github.com/greenelab/connectivity-search-backend/blob/af12f8cf2ad47d9a25ce8d1b7889390654eb3bb9/dj_hetmech_app/models.py "models.py for the dj_hetmech_app") Django's object-relational mapping framework (Figure @fig:database).
+We [import](https://github.com/greenelab/connectivity-search-backend/blob/af12f8cf2ad47d9a25ce8d1b7889390654eb3bb9/dj_hetmech_app/management/commands/populate_database.py "dj_hetmech_app populate_database management command") the data into a PostgreSQL database.
+Populating the database for all 2,205 metapaths up to length 3 was a prolonged operation, [taking](https://github.com/greenelab/connectivity-search-backend/pull/41#issuecomment-488054789) over 3 days.
+The majority of the time is spent populating the `DegreeGroupedPermutation` (37,905,389 rows) and `PathCount` (174,986,768 rows) tables.
+
+![
+**Schema for the connectivity search backend relational database models.**
+Each Django model is represented as a table,
+whose rows list the model's field names and types.
+Each model corresponds to a database table.
```

Comment:
This line seems redundant, because it is like a duplicate of line 426.

Reply:
table in 426 is referring to the visualization. 248 is saying django models have a 1-to-1 correspondence to PostgreSQL tables.